### PR TITLE
540 fix node version

### PR DESCRIPTION
**Pull request by sskular-wonderlab**
## Proposed Changes
Update Node version to 20.0.0

## Description
Fixes: https://github.com/cloudevents/sdk-javascript/issues/540

### CODE REVIEW

**Review by lance — APPROVED** [review]
/lgtm

File: package.json [hunk 1]
```diff
@@ -158,6 +158,6 @@
   },
   "types": "./dist/index.d.ts",
   "engines": {
-    "node": ">=12 <20.0.0"
+    "node": ">=12 <=20.0.0"
```
Comment (w3nl) [inline package.json:161]: Isn't it better to change it to `<=20`
Otherwise you say it's only working on 20.0.0, but not in 20.1.0 and 20.2.0, etc.

Comment (lance) [reply package.json:161]: Yes - that was addressed in #545 

https://github.com/cloudevents/sdk-javascript/blob/11442d32d307a0e8416ed573ce34fc825d3b63c4/package.json#L163

Comment (lance) [reply package.json:161]: Oh I see now... gotcha

**Pull request merged**